docs/SPEC.md

```diff
@@ -466,6 +466,12 @@ Examples:
     2018-07-01T00:00:00Z + 2y  // 2020-07-01T00:00:00Z
     2018-07-01T00:00:00Z + 5h  // 2018-07-01T05:00:00Z
 
+#### Binary types
+
+A _bytes type_ represents a sequence of byte values.
+The bytes type name is `bytes`.
+The bytes type is nullable.
```

Comment:
Why is a `bytes` type _nullable_? Is `bytes` just an alias for `[]byte`? Arrays are not _nullable_.